data/piper.1.in

```diff
@@ -0,0 +1,27 @@
+.\" Generated with help2man.
```

Comment:
Please generate this in meson instead of committing the generated file.

Reply:
That's impossible. It's generated _with_ help2man, but I edited lots of stuff to make it prettier. You can take a look at the generated man page with:
```sh
help2man builddir/piper.devel --no-discard-stderr > /tmp/piper.1
man -l /tmp/piper.1
```
```